### ISSUE TITLE
Cannot handle Date Column

### ISSUE THREAD

**Issue by MesumRaza** [PablocFonseca/streamlit-aggrid#3]
![image](https://user-images.githubusercontent.com/22402962/104934382-e16a3800-59cb-11eb-9ee2-78ba7a1179a4.png)

Cannot handle date column

df['Date']=pd.to_datetime(df['Date']).dt.date

**Comment by PablocFonseca** [comment]
Could you please post the code you're using here? 
This way I can reproduce the issue locally and fix it.

**Comment by PablocFonseca** [comment]
I think I found the error, I'll update a patch soon, meanwhile cast it to Datetime formats as a workaround

**Comment by MesumRaza** [comment]
Thankyou. Can you also please post some tutorials how to color code columns? some of links are broken on various webs.

**Comment by PablocFonseca** [comment]
This isssue is fixed in v0.1.6. I also add imedeltas support. 
Later this week I'll upload a more comple example with color code on columns. Meanwhile you can check on [ag-grid docs](https://www.ag-grid.com/documentation/javascript/cell-styles/)

To use with this component you just call (as line 113 in examples/example.py):

````
gb.configure_column("group",cellStyle=cellsytle_jscode)
```
